iohk/NixOps.hs

```diff
@@ -585,8 +574,14 @@ computeFinalDeploymentArgs o@Options{..} NixopsConfig{..} = do
 
 modify :: Options -> NixopsConfig -> IO ()
 modify o@Options{..} c@NixopsConfig{..} = do
-  printf ("Syncing Nix->state for deployment "%s%"\n") $ fromNixopsDepl cName
-  nixops o c "modify" $ Arg <$> cFiles
+  deplExists <- exists o c
```

Comment:
Why change this? Create is there for ages and is idempotent. Deploy should also modify before deploying, so no need to need modify. Also one would expect create to exist, not having to use modify for creation.

We could make create and modify just aliases.

Reply:
Ok, will do.

Reply:
Done.